### ISSUE TITLE
create new board fails because of permissions

### ISSUE THREAD

**Issue by ChristopherBarrington** [mheap/trello-cli#101]
Hi, Using the below command I am unable to create a new board and cannot understand the error message. I am able to add new cards to existing boards though.

```bash
> trello add-board -b bash -d "from bash"
/path/to/.conda/envs/trello/lib/node_modules/trello-cli/src/commands/add-board.js:44
        throw err;
        ^

Error: invalid value for prefs_permissionLevel
    at Request._callback (/path/to/.conda/envs/trello/lib/node_modules/trello-cli/node_modules/node-trello/lib/node-trello.js:88:13)
    at Request.self.callback (/path/to/.conda/envs/trello/lib/node_modules/trello-cli/node_modules/request/request.js:187:22)
    at Request.emit (events.js:315:20)
    at Request.<anonymous> (/path/to/.conda/envs/trello/lib/node_modules/trello-cli/node_modules/request/request.js:1044:10)
    at Request.emit (events.js:315:20)
    at IncomingMessage.<anonymous> (/path/to/.conda/envs/trello/lib/node_modules/trello-cli/node_modules/request/request.js:965:12)
    at IncomingMessage.emit (events.js:327:22)
    at endReadableNT (internal/streams/readable.js:1327:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  statusCode: 400,
  responseBody: 'invalid value for prefs_permissionLevel',
  statusMessage: 'Bad Request'
}
```

Is there any other information I can provide to help fix this? Or pointers of what I should do? 

Thanks,

**Comment by Aloysb** [comment]
Same here. Have you found a solution
?

**Comment by ChristopherBarrington** [comment]
Unfortunately not, though I haven't spent more time on it since opening the issue.

**Comment by mheap** [comment]
I've just published `v0.5.1` with a fix for this. The Trello API changed to require a permissions level (we previously didn't set it to use the default).

If your existing install, you can set it manually by running:

```
trello add-board -b bash -d "from bash" -p private
```

Alternatively, upgrade to `0.5.1` which has a default permission level of `private`

**Comment by Aloysb** [comment]
Legend thank you Michael.
